.github/workflows/github-issues/issues-labeled.yml

```diff
@@ -0,0 +1,22 @@
+name: When labels are modified, run actions.
```

Comment:
LGTM except can we get a accurate name on this (the name of the PR would probably work great)?

Reply:
It's setup to be modified for other label changes

Reply:
Isn't it better for other label actions to be splitted across multiple files or be all put in one file?

Reply:
I was making the files by event type, hence the file naming scheme